src/data/ACTIONS/NIN.js

```diff
@@ -31,122 +31,199 @@ export default {
 		},
 	},
 
-	// -----
-	// Player OGCDs
-	// -----
-
 	TEN: {
 		id: 2259,
 		name: 'Ten',
 		icon: 'https://xivapi.com/i/002000/002901.png',
-		onGcd: false,
-		cooldown: 0.5,
+		onGcd: true,
+		cooldown: 20,
+		gcdRecast: 0.5,
 	},
 
 	CHI: {
-		id: 2261,
+		id: 18806,
 		name: 'Chi',
 		icon: 'https://xivapi.com/i/002000/002902.png',
-		onGcd: false,
-		cooldown: 0.5,
+		onGcd: true,
+		cooldown: 20,
+		gcdRecast: 0.5,
 	},
 
 	JIN: {
-		id: 2263,
+		id: 18807,
 		name: 'Jin',
 		icon: 'https://xivapi.com/i/002000/002903.png',
-		onGcd: false,
-		cooldown: 0.5,
+		onGcd: true,
+		cooldown: 20,
+		gcdRecast: 0.5,
 	},
 
 	NINJUTSU: {
 		id: 2260,
 		name: 'Ninjutsu',
 		icon: 'https://xivapi.com/i/002000/002904.png',
-		onGcd: false,
+		onGcd: true,
 		cooldown: 20,
+		gcdRecast: 1.5,
 	},
```

Comment:
nit: Ninjutsu (any of them) do not technically have a cooldown anymore. It was moved to the mudras.

Reply:
Valid point - I'll correct the data.